design/src/risk-assessment.md

```diff
@@ -0,0 +1,70 @@
+# Risk Assessment
+
+This page keeps track of risks that we might face during development so that we can mitigate or even eliminate them.
+
+## Organizational Risks
+
+### Feature Creep
+
+[_Feature creep_](https://en.wikipedia.org/wiki/Feature_creep) refers to the continuos addition of features that goes way beyond the initially planned scope of the game.
+We are quite vulnerable to this for multiple reasons:
+
+- The game genre and theme invites for many complex features and mechanics that can be combined freely.
+  It's very easy to come up with additional ideas that would fit well into the game.
+
+- As an open source project, we are going to have many different contributors with different backgrounds.
+  Everyone might have their own ideas that they want to integrate into the game.
+
+To mitigate this risk, the following will be important:
+
+- Clearly document the vision of the game in the design book.
+  This will help to align all contributors on what we want to build.
+
+- Focus on the core game loop first and then iteratively plan the next set of features we want to add.
```

Comment:
Another thing we should do is create Github projects/milestones that closely align with the vision laid out in the design book.